var/spack/repos/builtin/packages/py-pytest/package.py

```diff
@@ -49,6 +50,7 @@ class PyPytest(PythonPackage):
     depends_on('py-attrs@17.2.0:', when='@3.3:3.4', type=('build', 'run'))
     depends_on('py-iniconfig', when='@6.0:', type=('build', 'run'))
     depends_on('py-packaging', when='@4.6:', type=('build', 'run'))
+    depends_on('py-pluggy@0.12:1', when='@6.2:', type=('build', 'run'))
     depends_on('py-pluggy@0.12:0', when='@4.6:', type=('build', 'run'))
     depends_on('py-pluggy@0.9.0:0.9,0.11:0', when='@4.5.0:4.5', type=('build', 'run'))
     depends_on('py-pluggy@0.11:', when='@4.4.2:4.4', type=('build', 'run'))
```

Comment:
The `win32` below should be `windows` if you don't mind changing it

Reply:
Sure